src/python/test/test_nextflow.py

```diff
@@ -92,12 +81,12 @@ def test_dxapp(self):
         [input1, input2]
     ])
     def test_dxapp_custom_input(self, *inputs):
+        with open(os.path.join(str(get_template_dir()), 'dxapp.json'), 'r') as f:
+            default_dxapp = json.load(f)
```

Comment:
instead of using `default_input_len = 7`, compare the default and the merged inputSpec dict directly